src/web/src/browser/views/styles/index.scss

```diff
@@ -8,6 +8,18 @@
 
 @import 'highlight.js/scss/github.scss';
 
+:root {
+  --theme-light-cyan: #0a90b7;
+  --theme-dark-cyan: #1889bc;
+  --theme-light-blue: #1a4480;
+  --theme-dark-blue: #162e51;
+  --theme-cornflower: #d9f1fd;
+  --theme-red: #c71f25;
+  --theme-light-grey: #f2f2f2;
+  --theme-grey-text: #757575;
+  --theme-dark-text: #5a5353;
+}
+
```

Comment:
I wonder if colors could be added to the USWDS theme, or assigned a token with the design system? Currently, the theme just overrides a couple values from the default theme: 
 https://github.com/18F/fedramp-automation/blob/master/src/web/src/browser/views/styles/_uswds-theme-overrides.scss#L12

Reply:
**Good News:** some of the colors from FedRamp are already part of the design system so I can swap those out.
<img width="558" alt="Screen Shot 2022-06-21 at 9 56 25 AM" src="https://user-images.githubusercontent.com/4629398/174831938-9a4ccc16-0cb0-4e0b-8563-69bc930cdee2.png">

**Bad News:** seems like you can only customize theme color tokens with existing uswds colors.

For example, I tried to replace the theme red with FedRamp red and it gave me an error.
`$theme-color-secondary-vivid: '#c71f25'`

<img width="1152" alt="Screen Shot 2022-06-21 at 10 03 31 AM" src="https://user-images.githubusercontent.com/4629398/174833075-07262fc9-060e-4271-9f27-32a86a536518.png">